docarray/utils/create.py

```diff
@@ -0,0 +1,220 @@
+from docarray import DocList, BaseDoc
```

Comment:
I would call the file `create_doc_class.py` or something like that (and make the functions directly importable from `docarray.utils`)

Reply:
I will change the file name, to make them importable, I would handle in another PR where all `utils` are exposed